### ISSUE TITLE
Dynamis entry not working

### ISSUE THREAD

**Issue by TeoTwawki** [DerpyProjectGroup/topaz#986]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened
- [x] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I have specified what branch this happens on branch: Base

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 
Updated my server over the weekend. Now nobody can enter Dynamis. I know there were some changes and thought maybe my players have outdated variable info or something since I was nearly 3 months behind.. But I just took a GM up to rank 6 from scratch and got the intro cs and keyitems, then went to Windurst walls and while I got the initial CS there it did not allow me to actually enter Dynamis. The trail markings simply say the default message of "the strands of grass here have been tied together".

No errors in server log.

**Comment by Xaver-DaRed** [comment]
Just to make sure, you have both Vial and Hourglass Key items and are over lvl 65?

**Comment by claywar** [comment]
Can you provide the event ID that you're hitting, will make a quicker trace

**Comment by TeoTwawki** [comment]
I think the scene intended to provide the hourglass _did not_ and I will double check that after I eat

**Comment by TeoTwawki** [comment]
Ok so remember that problem I discussed privately on discord where sometimes a cut scene did not "count" the 1st time? No prismatic hourglass. Didn't count even though the scene played. Now that I went back and redid it, I have one.

My existing non GM players, never went and got one because the old system was letting them in without one and when I pulled updates that got **fixed**. So players come to me telling me they can't get in today and that they could before maintenance happened... And they never had the hourglass because till now they never even needed it.
